### ISSUE TITLE
Reviews problem

### ISSUE THREAD

**Issue by Martzy303** [Tetrax-10/imdb-dark-mode#1]
I just installed this and it does not work on the reviews page, but I see a picture here that it should work.

![image](https://github.com/user-attachments/assets/44467eaf-cbf6-4d19-924d-decf91d630f0)

**Comment by Martzy303** [comment]
@Tetrax-10 If you could have a look. Thanks!

**Comment by Tetrax-10** [comment]
Yeah, I didn't add support for old UI. As it got totally different CSS variables and IMDb is replacing its old UI with new UI.

I will try to add support, when I got time.

**Comment by Martzy303** [comment]
@Tetrax-10 And does it work for the new UI or is there a new UI for reviews? If yes, how can I activate it?
In Content Setting I have disabled "Show reference view with full cast and crew (advanced view)"

**Comment by Tetrax-10** [comment]
IMDb doesn't have a new UI for review page. I hope they will implement it soon.
